metrics/report/example.toml

```diff
@@ -0,0 +1,23 @@
+# [mail] section specifies information for
+# a SMTP server authentication as well as the
+# sender and receipts of the report email.
+# This is in order to send the obtained results
+# from checktool to an email whitelist.
+[mail]
+smtp = "myserver.smtp.com"
+port = "25"
+user = ""
+identity = ""
+pass = ""
```

Comment:
A little tricky...
We won't be exposing any passwords on the github repo (just to dismiss any concern there), as config files will be site and server specific, so we will just host examples in the actual code base.
Having said that, sure, a non-plain text option would be nice - but, what options do we have?:
 - this is for SMTP - which ultimately wants a plain text string I believe. Even if there are secure alternatives, we probably need to support vanilla SMTP by default.
 - Does anybody have any other ideas? We could have links to an encrypted password and the relevant keys, but that still leaves you with the problem that anybody who can get to the config file can probably also get to the decryption key.

Ultimately I think this may come down to 'the system owner has to ensure they secure that file enough' via permissions. I'm all open to other suggestions, using a keyring API etc. etc.